### DISCUSSION TITLE
Keystrokes for Generate and Apply?

### DISCUSSION THREAD

**Discussion by Jakoebart** [Acly/krita-ai-diffusion#426] [Q&A] [answered]
Is there a way to use the "generate" and "apply" buttons with the keyboard? Thank you

**Comment by Acly** [comment, marked as answer]
Yes you can set up hotkeys in Krita's Keyboard Shortcut settings (look for Scripts > AI Tools: Image Generation)

**Reply by Tamulur** [reply]
Could you also add a shortcut for showing/hiding the preview? Right now to toggle the preview repeatedly (for comparison), I click the generated thumbnail. But often that starts the "selection drag" and the preview isn't toggled, and sometimes it's interpreted as double-click and applied instead of toggled.
